README.md

```diff
@@ -120,131 +168,156 @@ Pre-requisites
+
+One advantage of using `skip` authentication in development mode is that any user email can be entered without a password. Developers can use one of the emails that they loaded test data for in step (3) above. So if the test data loaded was with `-u shankari@eecs.berkeley.edu`, then the login email for the phone app would also be `shankari@eecs.berkeley.edu`.
+
+## 5. Beta-testing debugging
 If users run into problems, they have the ability to email logs to the
 maintainer. These logs are in the form of an sqlite3 database, so they have to
 be opened using `sqlite3`. Alternatively, you can export it to a csv with
 dates using the `bin/csv_export_add_date.py` script.
 
 ```
 <download the log file>
-$ mv ~/Downloads/loggerDB /tmp/logger.<issue>
-$ pwd
+mv ~/Downloads/loggerDB /tmp/logger.<issue>
+pwd
 .../e-mission-phone
-$ python bin/csv_export_add_date.py /tmp/loggerDB.<issue>
-$ less /tmp/loggerDB.<issue>.withdate.log
+python bin/csv_export_add_date.py /tmp/loggerDB.<issue>
+less /tmp/loggerDB.<issue>.withdate.log
 ```
 
-Contributing
----
-
-Add the main repo as upstream
-
-    $ git remote add upstream https://github.com/covid19database/phone-app.git
+## 6. Contributing
 
-Create a new branch (IMPORTANT). Please do not submit pull requests from master
 
-    $ git checkout -b mybranch
-
-Make changes to the branch and commit them
-
-    $ git commit
-
-Push the changes to your local fork
-
-    $ git push origin mybranch
-
-Generate a pull request from the UI
+1. Add the main repo as upstream
+```
+2. git remote add upstream <link/SSH key...>
+```
+3. Create a new branch (IMPORTANT). Please do not submit pull requests from master
+```
+4. git checkout -b <branch>
+```
+5. Make changes to the branch and commit them
+```
+6. git commit
+```
+ 7. Push the changes to your local fork
+```
+8. git push origin <branch>
+```
+9. Generate a pull request from the UI
 
-Address my review comments
+__\*__Address my review comments__\*__
 
 Once I merge the pull request, pull the changes to your fork and delete the branch
 ```
-$ git checkout master
-$ git pull upstream master
-$ git push origin master
-$ git branch -d mybranch
+git checkout master
+```
+```
+git pull upstream master
+```
+```
+git push origin master
+```
 ```
+git branch -d <branch>
+```
+
+---
+### Troubleshooting
+1. Xcode command line tools
+```
+Warning: No developer tools installed.
+You should install the Command Line Tools.
+```
+```
+xcode-select --install
+```
+
+2. Creating Logos
```

Comment:
why is this here?

Reply:
> why is this here?

From my experience upgrading from Monterey to Ventura (on two devices), I noticed that the update removes the CLI tools. Given that this has nothing to do with e-mission-phone and more to do with Mac devices, I can remove it in the next PR. I only added this because ease I remember you mentioning something regarding Mac specific and thought it would be useful.